routes/truckRoutes.js

```diff
@@ -1,3 +1,87 @@
-// connect to mongodb in this module as this is where you'll be making creat/read/delete calls to your database
+// connect to mongodb in this module as this is where you'll be making create/read/delete calls to your database
 // use 'mongodb://localhost/foodTruckAPI' for your mongoose connection string
 // remember this is a Node module
+
+var express = require('express');
+var mongoose = require('mongoose');
+//var bodyParser = require('body-parser');
+//var trucks = require('./trucks');
+var Truck = require('./models/truckModel');

```

Comment:
The relative path from this file to the truckModel module is '../models/truckModel'